Mathlib/LinearAlgebra/Dual.lean

```diff
@@ -671,6 +671,45 @@ theorem exists_dual_map_eq_bot_of_lt_top (hp : p < ⊤) (hp' : Free R (M ⧸ p))
   obtain ⟨f, hf, hf'⟩ := p.exists_dual_map_eq_bot_of_nmem hx hp'
   exact ⟨f, by aesop, hf'⟩
 
+variable {𝕜 E : Type*} [Field 𝕜] [AddCommGroup E] [Module 𝕜 E]
+
+open LinearMap Set FiniteDimensional
+
+private lemma mem_span_of_iInf_ker_le_ker' [FiniteDimensional 𝕜 E]
+    {n : ℕ} {L : Fin n → E →ₗ[𝕜] 𝕜} {K : E →ₗ[𝕜] 𝕜}
```

Comment:
Instead of indexing by `Fin n`, can you index by an arbitrary fintype (or, even better, by an arbitrary finite type)?